mpmath dependency: Fixed 'sys' NameError

lokeshh opened the pull request:
Fixed unwanted 'sys' NameError when mpmath dependency is not met 
by placing 'import sys' before mpmath dependency check.

Commit:
Dependencies: Fixed unwanted 'sys' NameError while mpmath dependency not met

Placed 'import sys' at the right place so as to facilitate sys.exit(-1)
without envoking invalid 'sys' Namerror when mpmath dependency is not met

asmeurer commented:
+1

toolforger commented on setup.py:
```diff
@@ -29,6 +29,13 @@
+import shutil
+import glob
+
+PY3 = sys.version_info[0] > 2
```
This should stay below with the "right Python version" part, below.

Commit:
Placed PY3 assigment with 'right Python version' check

Inorder to place 'import sys' at the right place I misplaced
assignment of PY3 which was supposed to reside with 'right
Python version'. I am placing PY3 assignment at the correct
place now.

smichr commented:
Thanks! Is there a name that we can add to the AUTHORS for you? I don't believe you are listed there yet.

lokeshh commented:
Yeah sure, its "Lokesh Sharma" (username: lokeshh).

I believe I hadn't configured the git properly, probably that's why it didn't showed any name up there. That has been sorted out best to my knowledge of git and hopefully it would show my name in future.

Thanks for the wonderful [Development Workflow](https://github.com/sympy/sympy/wiki/Development-workflow) guide you guys have created. It lead me to set up git within a day and help me started in such a short time.

smichr commented:
I'm glad the workflow helped. I recall how much @certik's suggestions for .config aliases helped me, and watching the debug session also helped to encourage efficient work habits, too.

smichr commented:
> Yeah sure, its "Lokesh Sharma" (username: lokeshh).

Should I use the `your_email@youremail.com` as the mailing address or was that part of the mistaken setup?

certik commented:
Thanks for the fix @lokeshh and thanks for merging @smichr. I am glad the videos were useful.

lokeshh commented:
No, kindly use `lokeshhsharma@gmail.com` as the mailing address. You are right, `your_email@youremail.com` was part of the mistaken setup.
